packages/core/src/anchor/memory/in-memory-anchor-service.ts

```diff
@@ -39,7 +39,12 @@ interface InMemoryAnchorConfig {
   verifySignatures?: boolean
 }
 
-const SAMPLE_ETH_TX_HASH = 'bagjqcgzaday6dzalvmy5ady2m5a5legq5zrbsnlxfc2bfxej532ds7htpova'
+// Caches recent anchor txn hashes and the timestamp when they were anchored
+// This is intentionally global and not a member of InMemoryAnchorService. This is so that when
```

Comment:
I'm not thrilled about this, but it was the only way to get the tests passing.  It does kind of make the InMemoryAnchorService work more like the real anchor service, so it kind of makes sense.  Probably a better solution would be to make tests that use multiple Ceramic nodes share the same InMemoryAnchorService instance, but that would be a bigger change.